bundles/org.openhab.binding.solax/README.md

```diff
@@ -12,6 +12,9 @@ In case the parsed information that comes with the binding out of the box differ
 
 ## Supported Things
 
+Note: Channels may vary depending on the invertor type and the availability of information for parsing the raw data. 
```

Comment:
```suggestion
Note: Channels may vary depending on the inverter type and the availability of information for parsing the raw data. 
```

Also, consider if this should be moved below the table, just before the channels are being listed? This way the supported Thing would be introduced before this specific note about the availability of that Thing's channels.